aquacomputer_d5next.c

```diff
@@ -1152,13 +1243,44 @@ static int aqc_write(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 							ctrl_values_sizes, 3);
 				if (ret < 0)
 					return ret;
-			} else {
+				break;
+			case aquastreamxt:
+				if (channel == 0) {
+					/* Pump speed can be controlled between 3000 and 6000 rpm */
+					pwm_value = DIV_ROUND_CLOSEST(val * 50, 255) * 60 + 3000;
```

Comment:
For the life of me I can't decipher what *50 and *60 do. Perhaps it's something simple, but please note what those values are.